### PR TITLE
remove SIT as codeowner of integrations omnibus files

### PR DESCRIPTION
**Pull request by cedricvanrompay-datadog**
### What does this PR do?

remove SIT as codeowner of integrations omnibus files

### Motivation

This ownership does not seem necessary and causes the integrations team to be waiting on trivial reviews from SIT on a regular manner

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by FlorentClarret — APPROVED** [review]

**Review by ethan-lowman-dd — APPROVED** [review]

**Review by jamessteel123 — APPROVED** [review]

**Review by KevinFairise2 — APPROVED** [review]
Your team have been added to Codeowners 1 year ago to follow the modifications on the files(cf [PR](https://github.com/DataDog/datadog-agent/pull/10844)). If you do not need it anymore, LGTM

**Pull request merged**
